src/modules/sensors/camera.c

```diff
@@ -290,18 +309,55 @@ static void recv_frame(int i) {
     state.brightness[i] = compute_brightness(buf.bytesused) / CAMERA_ILL_MAX;
 }
 
+static void reset_histogram(void) {
+    state.hist.count = memset(state.hist.count, 0, HISTOGRAM_STEPS * sizeof(double));
```

Comment:
We  can only memset state.hist here then. (`HISTOGRAM_STEPS * sizeof(struct histogram)`)